ui/components/app/asset-list/asset-list.js

```diff
@@ -155,8 +166,13 @@ const AssetList = ({ onClickAsset }) => {
               secondaryCurrencyProperties.value
             }
             tokenSymbol={primaryCurrencyProperties.suffix}
-            secondary={showFiat ? secondaryCurrencyDisplay : undefined}
+            secondary={
+              isTestNet || (showFiat && isOriginalNativeSymbol)
+                ? secondaryCurrencyDisplay
+                : undefined
+            }
```

Comment:
hey @salimtb , why did we add the `isTestnet `here?

Reply:
hey @sahar-fehri ,
because we want to display the conversion for `testnet` , even if the symbol is ETH for example.